src/scss/attendee/layout.scss

```diff
@@ -3,6 +3,7 @@
 $w-sidebar: 416px; // previously named $sidebar-width
 $h-video-chat: 160px; // previously named $video-chat-height
 
+$w-overlay--med: 715px;
 $w-overlay--max: 800px;
```

Comment:
[Q] Aren't these `$w-overlay` values supposed to somehow follow those in `"scss/attendee/breakpoint";` ?

Reply:
these are a little different and these don't align with breakpoints 1:1, it's because different overlay tab content needs different minimal/max width allocated to be displayed correctly, so having these for now is alright